src/shared/components/Button/Button.style.ts

```diff
@@ -103,25 +103,16 @@ const sizeFromProps = ({ size = 'regular', full, hasText }: ButtonStyleProps) =>
 const disabled = ({ disabled }: ButtonStyleProps) =>
   disabled
     ? css`
-        box-shadow: none;
-        color: ${colors.gray[200]};
-        background-color: ${colors.gray[400]};
-        border-color: ${colors.gray[400]};
-        &:hover {
-          color: ${colors.gray[200]};
-          background-color: ${colors.gray[400]};
-          border-color: ${colors.gray[400]};
-        }
-        &:active {
-          color: ${colors.gray[200]};
-          background-color: ${colors.gray[400]};
-          border-color: ${colors.gray[400]};
-        }
+        filter: brightness(50%);
```

Comment:
This is a nice shortcut, but it will not produce the values specified by the design, I think we should implement what's in there.